### PR TITLE
Documentation/add pages

### PR DESCRIPTION
**Pull request by capriceesmas**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Explore the distributed web
 [Explore](./explore)
 [Blog](./blog/)
 [Protocols](./protocols)
+[Docs](./docs/index)
```
Comment (RangerMauve) [inline index.md:30]: With the way index.md works, you can just link to `/docs` and it'll get loaded automagically

```suggestion
[Docs](./docs/)
```

**Review by RangerMauve — COMMENTED** [review]
Good start!

Would you be down to write some sort of blurb in the index along the lines of "This is a work in progress" and "These docs can help you get started with making apps using Agregore"?

File: index.md (hunk 1 above)
Comment (capriceesmas) [reply index.md]: Made the changes you asked for! Hopefully you enjoy my emoji usage

**Review by RangerMauve — APPROVED** [review]
LGTM!

**Pull request merged**